dataRetriever/resolvers/headerResolver.go

```diff
@@ -114,10 +114,7 @@ func (hdrRes *HeaderResolver) ProcessReceivedMessage(message p2p.MessageP2P, fro
 		return err
 	}
 
-	err = hdrRes.antifloodHandler.CanProcessMessageOnTopic(fromConnectedPeer, hdrRes.topicNameForAntiflood())
-	if err != nil {
-		return err
-	}
+	//TODO add antiflooding capabilities on resolvers
```

Comment:
so every header is accepted right now ? no antiflood ?

Reply:
no, the purpose of the PR was to throttle the incoming stream of headers, consensus messages and heartbeat messages. The requests filtering will be added in a next PR